### ISSUE TITLE
Release v1.49.6

### ISSUE THREAD

**Issue by deckhouse-BOaTswain** [deckhouse/deckhouse#5573]
:robot: A dedicated issue to run tests and deploy release [v1.49.6](https://github.com/deckhouse/deckhouse/milestone/219).

---

<details>
<summary>Release issue commands and options</summary>
<br />

You can trigger release related actions by commenting on this issue:

- `/deploy/<channel>[/<editions>]` will publish built images into the release channel.
  - `channel` is one of `alpha | beta | early-access | stable | rock-solid`
  - `editions` is a comma-separated list of editions `ce | ee | fe`
- `/suspend/<channel>` will suspend released version.
  - `channel` is one of `alpha | beta | early-access | stable | rock-solid`
- `/e2e/run/<provider> git_ref` will run e2e using provider and an `install` image built from git_ref.
  - `provider` is one of `aws | azure | eks | gcp | openstack | static | vsphere | yandex-cloud`
  - `git_ref` is a tag `v1.49.6 | test-v1.49.6*` or a branch `main | release-1.49`
- `/e2e/run/<provider> git_ref_1 git_ref_2` will run e2e using provider and git_ref_1 and then switch to git_ref_2.
  - `provider` is one of `aws | azure | eks | gcp | openstack | static | vsphere | yandex-cloud`
  - `git_ref_1` is a release-* or main branch
  - `git_ref_2` is a release-* or main branch
- `/e2e/use/cri/<cri_name>` specifies which CRI to use for e2e test.
  - `cri_name` is one of `docker | containerd`
- `/e2e/use/k8s/<version>` specifies which Kubernetes version to use for e2e test.
  - `version` is one of `1.23 | 1.24 | 1.25 | 1.26 | 1.27`
- `/build git_ref` will run build for release related refs.
  - `git_ref` is a tag `v1.49.6 | test-v1.49.6*` or a branch `main | release-1.49`

**Note 1:**
A single command `/e2e/run/<provider>` will run e2e with default CRI 'containerd' and Kubernetes version '1.25'.
Put `/e2e/use` options below `/e2e/run` command to set specific CRI and Kubernetes version. E.g.:

```
/e2e/run/aws main
/e2e/use/cri/docker
/e2e/use/cri/containerd
/e2e/use/k8s/1.20
/e2e/use/k8s/1.23

This comment will run 4 e2e jobs on AWS with Docker and containerd
and with Kubernetes version 1.20 and 1.23 using image built from main branch.
```

```
/e2e/run/aws release-1.35 release-1.36
/e2e/use/cri/containerd
/e2e/use/k8s/1.23

This comment will create cluster in AWS using Deckhouse built from release-1.35 branch
and then switch to images built from release-1.36 branch.
'use' options are supported, cluster will use containerd and Kubernetes version 1.23.
```

**Note 2:**
'deploy', 'suspend' and 'e2e' commands should run after 'build FE' job is finished.

**Note 3:**
No autobuild for release branch. Run this command after cherry-picking into release branch:
```
/build release-1.49
```

</details>

**Comment by deckhouse-BOaTswain** [comment]
New tag **v1.49.6** is created.

**Comment by deckhouse-BOaTswain** [comment]
New commit [dc1b6b](https://github.com/deckhouse/deckhouse/commit/dc1b6b7bae1007f9e3dda651d482d5b380e3c9e1) in branch **main**:
```
[ci] Add GOPROXY env variable (#5579)

Signed-off-by: Nikolay Mitrofanov <nikolay.mitrofanov@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [cbb8ba](https://github.com/deckhouse/deckhouse/commit/cbb8ba403db029ed8e4605ac350b5a213ae3a2a6) in branch **main**:
```
Revert "[bashible] fixed wait apt update (#5314)" (#5580)
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [2a62f5](https://github.com/deckhouse/deckhouse/commit/2a62f5527233180e1e397b6a45cc81083c3aa02a) in branch **main**:
```
[getting started] Fix kind installation script (#5576)


Signed-off-by: Artem Kladov <artem.kladov@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [c340c1](https://github.com/deckhouse/deckhouse/commit/c340c17b3f8fe108c03e1a1fd3626b739d807cda) in branch **main**:
```
[ingress-nginx] add prestop lifecycle hook for rbac-proxy (#5370)

Signed-off-by: Mikhail Scherba <mikhail.scherba@flant.com>
Co-authored-by: Mikhail Scherba <mikhail.scherba@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [b0cadd](https://github.com/deckhouse/deckhouse/commit/b0cadd8cac108d657b2afa0faacaec0531ef4bb3) in branch **main**:
```
[user-authz] Refactoring ClusterAuthorizationRule limitNamespaces (#4666)

Signed-off-by: Mikhail Scherba <mikhail.scherba@flant.com>
Signed-off-by: Artem Kladov <6360800+z9r5@users.noreply.github.com>
Signed-off-by: Artem Kladov <artem.kladov@flant.com>
Signed-off-by: Mikhail Scherba <41360396+miklezzzz@users.noreply.github.com>
Signed-off-by: Maksim Nabokikh <maksim.nabokikh@flant.com>
Co-authored-by: Mikhail Scherba <mikhail.scherba@flant.com>
Co-authored-by: Artem Kladov <6360800+z9r5@users.noreply.github.com>
Co-authored-by: Artem Kladov <artem.kladov@flant.com>
Co-authored-by: Maksim Nabokikh <maksim.nabokikh@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [3eb221](https://github.com/deckhouse/deckhouse/commit/3eb221fb0670865ed487568637d28da952274ed0) in branch **main**:
```
[monitoring] add livenessProbe for node-exporter (#5558)

Signed-off-by: alexey.chazov <alexey.chazov@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [878392](https://github.com/deckhouse/deckhouse/commit/87839250b81722acbab521e4e366e39ba37b20d4) in branch **main**:
```
[readme] Update D8 certification information in the README.md (#5589)

Signed-off-by: Artem Kladov <6360800+z9r5@users.noreply.github.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [0c5743](https://github.com/deckhouse/deckhouse/commit/0c57430b7cc8be1ff0e5d0fbee18bb5c5ff8ed07) in branch **main**:
```
[flant-integration] Change `sum` to `avg` in `controller_metrics` hook and refactor utils (#5517)

Signed-off-by: alexey.makhonin <alexey.makhonin@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [fb15e4](https://github.com/deckhouse/deckhouse/commit/fb15e49d80ff50665bd085f041cccbb4a2656bf4) in branch **main**:
```
Change external modules to modules

Signed-off-by: Maksim Nabokikh <maksim.nabokikh@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [409508](https://github.com/deckhouse/deckhouse/commit/4095082dd4a22c01aa279212a2ed6915cf7d2d81) in branch **main**:
```
[external-module-manager] Fix race condition while handling source on Deckhouse startup (#5598)

Signed-off-by: m.nabokikh <maksim.nabokikh@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [70f764](https://github.com/deckhouse/deckhouse/commit/70f764fc97da5de6cc5896de6bcd5dd6b9ba9061) in branch **main**:
```
[documentation] Remove the unnecessary user directive from the nginx.conf (#5600)

Signed-off-by: Artem Kladov <artem.kladov@flant.com>main
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [056df1](https://github.com/deckhouse/deckhouse/commit/056df187269df24fe7507355b2206471612f8257) in branch **main**:
```
[global] Add .ModulePath to werf.inc.yaml (#5458)

Signed-off-by: Nikolay Mitrofanov <nikolay.mitrofanov@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [196b66](https://github.com/deckhouse/deckhouse/commit/196b66b109f849335ff0442046763c8427167b3c) in branch **main**:
```
[monitoring-applications] change expression to `process_virtual_memory`  (#5543)

Signed-off-by: kirill-kashin <90387970+kirill-kashin@users.noreply.github.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [ad6cc3](https://github.com/deckhouse/deckhouse/commit/ad6cc349ac5b49eab741212439e748e96731ca60) in branch **main**:
```
Re-generate changelog v1.50.2 (#5562)

Signed-off-by: deckhouse-BOaTswain <89150800+deckhouse-boatswain@users.noreply.github.com>
Co-authored-by: z9r5 <z9r5@users.noreply.github.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [05379b](https://github.com/deckhouse/deckhouse/commit/05379bc57175acd6a280ee54244f54507c749c4f) in branch **main**:
```
[registrypackages] Add .ModulePath to werf (#5610)

Signed-off-by: Nikolay Mitrofanov <nikolay.mitrofanov@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [519b54](https://github.com/deckhouse/deckhouse/commit/519b543d5407f9139cddbc233895059321a79b9d) in branch **main**:
```
[monitoring-kubernetes] Fix grafana dashboard for Ip_Forwarding (#5586)

Signed-off-by: Andrey Babushkin <fishday53@gmail.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [93a8db](https://github.com/deckhouse/deckhouse/commit/93a8dbff424152034074db89ca35853ba908b948) in branch **main**:
```
[distroless] log-shipper (#5523)

Signed-off-by: Alexey Sulimenko <alexey.sulimenko@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [03df71](https://github.com/deckhouse/deckhouse/commit/03df71011a591343c4b8abeb0b09351d3eace6ce) in branch **main**:
```
[distroless] extended-monitoring (#5547)

Signed-off-by: Alexey Sulimenko <alexey.sulimenko@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [298f29](https://github.com/deckhouse/deckhouse/commit/298f29269ac0360051b388237522c01b01bd3f7a) in branch **main**:
```
[prometheus-metrics-adapter] Distroless (#5519)

Signed-off-by: Andrey Klimentyev <andrey.klimentyev@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [6be75b](https://github.com/deckhouse/deckhouse/commit/6be75b107b5962bc516cb8e9d425b874f578d124) in branch **main**:
```
[docs] Updates the monitoring section of the documentation sidebar (#5611)

Signed-off-by: Artem Kladov <artem.kladov@flant.com>
Co-authored-by: Konstantin Aksenov <konstantin.aksenov@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [bbfb05](https://github.com/deckhouse/deckhouse/commit/bbfb055af597a0298eeeea724267c7b51760bb2b) in branch **main**:
```
[deckhouse] move ModuleConfig handling to the deckhouse module (#5566)

Signed-off-by: Yuriy Losev <yuriy.losev@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [2fc2f8](https://github.com/deckhouse/deckhouse/commit/2fc2f807f707e840f2df69e38cce306a9b5e004e) in branch **main**:
```
[deckhouse] Step by step update when deckhouse release version. (#5453)

Signed-off-by: dmitry.koba <dmitry.koba@flant.com>
Signed-off-by: Yuriy Losev <yuriy.losev@flant.com>
Co-authored-by: dmitry.koba <dmitry.koba@flant.com>
Co-authored-by: Yuriy Losev <yuriy.losev@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [2cfaa9](https://github.com/deckhouse/deckhouse/commit/2cfaa942eabb65cc19bb87a34f9eeb9e7847dc83) in branch **main**:
```
[candi] Remove cleanup-after-prune-containerd-content ngc (#5533)

Signed-off-by: Yan Zarytovsky <yan.zarytovsky@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [c72d04](https://github.com/deckhouse/deckhouse/commit/c72d04f545181730ead6dd0b407bc27c476c64b4) in branch **main**:
```
[getting started] Separate cards for the openstack-based installation (#5609)

Signed-off-by: Artem Kladov <artem.kladov@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [7f8871](https://github.com/deckhouse/deckhouse/commit/7f8871c9cf2a25126c2e3b263a3da50261c0fdc0) in branch **main**:
```
[deckhouse] Update FAQ documentation about minimalNotificationTime. (#5581)

Signed-off-by: dmitry.koba <dmitry.koba@flant.com>
Co-authored-by: Artem Kladov <artem.kladov@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [8a95cc](https://github.com/deckhouse/deckhouse/commit/8a95ccb641c252121818c63235ff7cdf0658498d) in branch **main**:
```
[getting started] Remove protocol prefix from the imagesRepo (#5617)

Signed-off-by: Artem Kladov <artem.kladov@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [0744c5](https://github.com/deckhouse/deckhouse/commit/0744c52c94da17dd67b70a1f28870e553f3c574b) in branch **main**:
```
[e2e] linstor tests in e2e static is temporary disabled (#5605)

Signed-off-by: v.oleynikov <vasily.oleynikov@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [e71d12](https://github.com/deckhouse/deckhouse/commit/e71d12732839d25d134a99cbce855c37416c53b8) in branch **main**:
```
[cloud-provider-openstack] added a command to get the list of networks (#5590)

Signed-off-by: alexey-gavrilov-flant <53515419+alexey-gavrilov-flant@users.noreply.github.com>
Co-authored-by: Denys Romanenko <65756796+RomanenkoDenys@users.noreply.github.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [7810c0](https://github.com/deckhouse/deckhouse/commit/7810c0ff1b00e510fdc0d6e8dc7255fabc29be11) in branch **main**:
```
[deckhouse]  Handle Endpoints and orphan EndpointSlices (#5620)

Signed-off-by: Yuriy Losev <yuriy.losev@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [23a77e](https://github.com/deckhouse/deckhouse/commit/23a77e9098b565e1a8fc29a18890ad2b33bbcfbb) in branch **main**:
```
[control-plane-manager] add a state check before changing the master node (#5487)


Signed-off-by: Alexey Gavrilov <alexey.gavrilov@flant.com>
Signed-off-by: Denys Romanenko <65756796+RomanenkoDenys@users.noreply.github.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [ee7798](https://github.com/deckhouse/deckhouse/commit/ee7798e8bf1fa813f27d6dc650f2d93c0fe1c32f) in branch **main**:
```
[runtime-audit-engine] move falcosidekick to distroless (#5623)

Signed-off-by: Denis Romanenko <denis.romanenko@flant.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [ec8ccc](https://github.com/deckhouse/deckhouse/commit/ec8cccf6355d2baccb004c86cfaec82e288dbae3) in branch **main**:
```
Fix docs (#5628)

Signed-off-by: Konstantin Kozhin <1387510+kozhin@users.noreply.github.com>
```

**Comment by deckhouse-BOaTswain** [comment]
New commit [c5cc0a](https://github.com/deckhouse/deckhouse/commit/c5cc0ab7375eccfe04b7828c9c9d977ce6e9b20a) in branch **main**:
```
[docs][extended-monitoring] Fix documentation (#5612)

Signed-off-by: maray2208 <113165676+maray2208@users.noreply.github.com>
```
